ivy/functional/backends/torch/experimental/statistical.py

```diff
@@ -6,6 +6,23 @@
 from ivy.func_wrapper import with_unsupported_dtypes
 from . import backend_version
 
+def histogramdd(
+        input: torch.tensor,
+        /,
+        *,
+        bins: Optional[Union[Tuple[int],int,torch.Tensor]] = 10,
+        range: Optional[Union[Tuple[int], int]] = [0., 1., 0., 1.],
+        weights : torch.tensor,
```

Comment:
You cannot define non-default arguments after default arguments. Please provide a default value for `weights`